### PR TITLE
37 change UI

### PR DESCRIPTION
**Pull request by annierusu**
xml based ui for the app. Please run the app to see how everything looks :)

### CODE REVIEW

File: app/src/main/res/layout/fragment_web_api.xml [hunk 1]
```diff
@@ -20,34 +20,52 @@
         app:layout_constraintStart_toStartOf="parent"
```
Comment (Cocograal) [inline app/src/main/res/layout/fragment_web_api.xml:20]: Same in this file, you should use a convention for the ids of the buttons

File: app/src/main/res/values-night/themes.xml [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
-        <item name="colorSecondary">@color/teal_200</item>
-        <item name="colorSecondaryVariant">@color/teal_700</item>
+        <item name="colorSecondary">@color/pink_200</item>
+        <item name="colorSecondaryVariant">@color/pink_900</item>
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
         <item name="android:statusBarColor">?attr/colorPrimaryVariant</item>
         <!-- Customize your theme here. -->
     </style>
-</resources>
```
Comment (Cocograal) [inline app/src/main/res/values-night/themes.xml:16]: line missing refer to my previous comment to make it automatic

**Review by Cocograal — CHANGES_REQUESTED** [review]
Overall the code is good, but make sure to use convention "fragmentName_name" and make the changes in the IDE so that the format is the same everywhere.

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -21,40 +21,45 @@
...
         app:layout_constraintHorizontal_bias="1.0"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.303" />
 
-    <Button
-        android:id="@+id/filterFragmentButton"
-        android:layout_width="wrap_content"
+    <TextView
+        android:id="@+id/text_home_text"
```
[the remark this answers is not in the document]
Comment (Cocograal) [reply app/src/main/res/layout/fragment_home.xml:39]: The PR has been closed but here's the link
https://github.com/Tarjetakuna/Tarjetakuna/pull/59

File: app/src/main/res/layout/fragment_sign_out.xml [hunk 4]
```diff
@@ -1,56 +1,36 @@
...
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toStartOf="parent" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
+        android:id="@+id/signOut_home_button"
+        style="@style/HomeButtonStyle"
+        android:text="@string/go_back_to_home_page"
+        tools:ignore="MissingConstraints" />
```
Comment (Cocograal) [inline app/src/main/res/layout/fragment_sign_out.xml:35]: Should we really use "ignore missingConstraints" here?

Comment (annierusu) [reply app/src/main/res/layout/fragment_sign_out.xml:35]: the style HomeButtonStyle imports the constraints (as our home buttons are all in the same place it's efficient to just define that elsewhere), it just complains if you dont explicity write contraints within the layout

**Review by JulienDiTria — APPROVED** [review]
good job for the refactoring of UI

**Review by Cocograal — APPROVED** [review]

File: app/src/main/java/com/github/sdp/tarjetakuna/ui/home/HomeViewModel.kt [hunk 5]
```diff
@@ -6,9 +6,14 @@ import androidx.lifecycle.ViewModel
...
 
-    private val _text = MutableLiveData<String>().apply {
-        value = "This is home Fragment"
+    private val _titleText = MutableLiveData<String>().apply {
+        value = "Welcome to Tarjetakuna!\n"
     }
-    val text: LiveData<String> = _text
+    private val _descriptionText = MutableLiveData<String>().apply {
+        value =
+            "Start browsing right away, or sign in to view and manage your Magic: The Gathering collection\n"
```
Comment (SolalPirelli) [inline app/src/main/java/com/github/sdp/tarjetakuna/ui/home/HomeViewModel.kt:14]: Use resource strings?

**Pull request merged**